models/yang/sonic/sonic-acl.yang

```diff
@@ -115,6 +115,8 @@ module sonic-acl {
 						enum FORWARD;
 						enum DROP;
 						enum REDIRECT;
+						enum int_insert;
```

Comment:
Any reason for lower case, if not lets maintain in upper case for consistency.

Reply:
No specific reason except the FS had it that way.